docs/pipelines.md

```diff
@@ -209,6 +209,36 @@ const pipeline = blueprints.CodePipelineStack.builder()
             .build(...)
 ```
 
+## Monorepo support
+
+In some cases, customers choose to have their bllueprints defined in subdirectories on a monorepo project. In this scenario, the root of the project is not the root of the CDK pipeline. A single monorepo can also support multiple subprojects each containing its own pipeline. 
```

Comment:
typo `bllueprints`

Reply:
`sub directories`

Reply:
it is actually subdirectory - one word. Blueprints is fixed.